ci.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 set -e
 RVM_PATH=/home/runner/.rvm
-ARGS=("$@")
+MATRIX_RUBY_VERSION=$1
+MATRIX_RAILS_VERSION=$2
 
 if [[ ! -d $RVM_PATH ]]; then
-  curl -sSL https://get.rvm.io | bash
+  curl -sSL https://get.rvm.io | bash -s stable --ruby
 fi
 
 source /home/runner/.rvm/scripts/rvm
```

Comment:
curious: using `act` I don't have this /home/runner directory, and RVM installs to /usr/local/.rvm

Reply:
I think I've figured this part out, it looks like locally `act` installs to `/usr/local/rvm` and I need to run `source /usr/local/rvm/scripts/rvm`

Reply:
environment on act is different from github CI. act installs in root, but github uses a user account